### PR TITLE
Add support for XRC chain

### PR DESCRIPTION
**Pull request by noahxrc**
Hello,
we added support for XRC (xrhodium.org) coin. 

Can you merge it please?

### CODE REVIEW

**Comment by noahxrc** [comment]
image
Everything seems to be working :)

File: angular/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -20,5 +20,5 @@ export const environment: IEnvironment = {
   instanceUrl: 'https://www.blockcore.net/',
   instanceExplorerUrl: 'https://explorer.blockcore.net',
   networks: [] as string[] // Returns all
-  // networks: ['STRAX', 'CRS', 'TSTRAX', 'TCRS', 'IMPLX', 'MOL', 'CITY', 'CY', 'RSC', 'SBC', 'X42']
+   //networks: ['STRAX', 'CRS', 'TSTRAX', 'TCRS', 'IMPLX', 'MOL', 'CITY', 'CY', 'RSC', 'SBC', 'X42','XRC']
```
Comment (sondreb) [inline angular/src/environments/environment.prod.ts:23]: Please remove this, no networks should be defined here, that will ensure all networks are always loaded.

**Review by sondreb — CHANGES_REQUESTED** [review]
Just one minor change and then everything else is OK. Thanks for the PR!

File: angular/src/environments/environment.prod.ts (hunk 1 above)
Comment (noahxrc) [reply angular/src/environments/environment.prod.ts]: Ok i removed whole line. :) Thank you :)

**Review by sondreb — APPROVED** [review]
Great work, thanks!

**Pull request merged**